packages/paste-website/src/components/shortcodes/table-of-contents/index.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import {Anchor} from '@twilio-paste/anchor';
+import {TableOfContentsProps} from './types';
+import {StyledWrapper, StyledScrollSpy, StyledListItem} from './styles';
+import {slugify} from '../../../utils/RouteUtils';
+
+const TableOfContents: React.FC<TableOfContentsProps> = headings => {
+  if (headings == null || headings.data == null) {
+    return null;
+  }
+
+  const allHeadings = headings.data;
```

Comment:
oh, so what I meant was change

```
<TableOfContents data={props.data.mdx.headings} />
```

to

```
<TableOfContents headings={props.data.mdx.headings} />
```

then you can destructure it from props

```
const TableOfContents: React.FC<TableOfContentsProps> = ({headings}) => {
  if (headings == null) {
    return null;
  }

  // Get Array of heading anchors. Excluding h1 elements.
  const headingsList = headings
    .filter(heading => heading.depth !== 1)
    .map(({value}) => {
      const headingAnchor = slugify(value);
      return headingAnchor;
    });
```

Reply:
👍 Got it.